presto-hive/src/test/java/com/facebook/presto/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -829,7 +829,9 @@ private void testCreatePartitionedBucketedTableAsFewRows(
 
         assertUpdate(
                 // make sure that we will get one file per bucket regardless of writer count configured
-                getTableWriteTestingSession(optimizedPartitionUpdateSerializationEnabled),
+                Session.builder(getTableWriteTestingSession(optimizedPartitionUpdateSerializationEnabled))
+                        .setCatalogSessionProperty(catalog, "create_empty_bucket_files", String.valueOf(optimizedPartitionUpdateSerializationEnabled))
```

Comment:
This is the wrong test parameter here. It's supposed to be `createEmpty` which wasn't added to the test method.

Reply:
Added. Thanks